Examples/ScalarField/ScalarFieldLevel.cpp

```diff
@@ -63,7 +63,8 @@ void ScalarFieldLevel::prePlotLevel()
     Potential potential(m_p.potential_params);
     ScalarFieldWithPotential scalar_field(potential);
     BoxLoops::loop(MatterConstraints<ScalarFieldWithPotential>(
-                       scalar_field, m_dx, m_p.G_Newton),
+                       scalar_field, m_dx, c_Ham, Interval(c_Mom, c_Mom), -1,
+                       Interval(), m_p.G_Newton),
```

Comment:
I would prefer having either another constructor without the arguments for the abs of the constraints OR to put G_Newton first, so that those default arguments of -1 and Interval() don't need to be put by hand

Reply:
I have made `G_Newton` an earlier argument in the constructor and removed its default value too.